src/patterns/find-a-support-centre/examples/results/index.njk

```diff
@@ -0,0 +1,130 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+
+{% from "components/collapsible/_macro.njk" import onsCollapsible %}
+
+{% set pageConfig = {
+    "title": "Census 2021",
+    "theme": "census",
+    "absoluteUrl": pageInfo.url,
+    "headMeta": {
+        "description": "Census 2021 is a UK wide population survey."
+    },
+    "header": {
+        "logoHref": "#",
+        "titleLogo": "census-logo-en",
+        "titleLogoHref": "#",
+        "titleLogoAlt": "Census 2021"
+    },
+    "language": {
+        "allLanguagesUrl": "#",
+        "languages": [
+            {
+                "url": "#",
+                "ISOCode": "en",
+                "text": "English",
+                "buttonAriaLabel": "Language selector. Current language: English",
+                "chooseLanguage": "Choose language",
+                "allLanguages": "All languages",
+                "current": true
+            },
+            {
+                "url": "#",
+                "ISOCode": "cy",
+                "text": "Cymraeg",
+                "buttonAriaLabel": "Dewisydd iaith. Iaith gyfredol: Cymraeg",
+                "chooseLanguage": "Dewiswch iaith",
+                "allLanguages": "Pob iaith",
+                "current": false
+            }
+        ]
+    }
+} %}
+
+{% block main %}
+
+    <h1 class="u-mt-l u-fs-xxl">Support centres near <mark class="highlight">S1 1XN</mark></h1>
+
+    <ul class="support-centre">
+        
+        <li class="support-centre__item">
+
+            <h2 class="u-fs-l u-mb-xs">Sheffield Central Library</h2>
+            <span class="u-fs-r u-mb-s u-p-xxs support-centre__distance">0.7 miles away</span>
+
+            <p>First Floor, Central Library <br> Surrey Street <br> Sheffield <br> S1 1XZ</p>
+
+            <p>Find us on <a class="icon--external-link" href="https://goo.gl/maps/X48tjZkXA4RaN3LL7" target="_blank" rel="noopener nofollow">Google Maps</a> <br> Telephone: 01142 734 712</p>
+
+            {% set centreDetails %}                          
+            <h2 class="u-fs-r--b u-mb-xs">Opening Times</h2>
+
+            <p>9am to 4pm - Monday to Friday <br> 9am to 5pm - Saturday to Sunday (Census weekend)</p>
+            
+            <h2 class="u-fs-r--b u-mb-xs">Centre facilities</h2>
+
+            <ul class="support-centre__list">
```

Comment:
Think we should use the list component here

Reply:
Unfortunately not. The items sit inside a `<ul>`.

So you get the `<ul>` inside a `<ul>` style. Ideally we would assign modifier classes for bullet styling like so:

```
<ul class="list list--bullet">
  <li class="list__item">Item 1</li>
</ul>
```

Reply:
I'm not sure I'd use a list at the higher level (this is always a much debated thing). If a list item contains multiple elements (headings, para's etc) and include another list that isn't related to its parent `li` I'm not convinced how much semantic meaning it provides. I think the headings provide the content hierarchy alone.

Reply:
The HTML structure has been changed. Agreed the heading structure alone should cater for this. As a result of this change the list item now renders correctly, although it would be worth looking at the list component styling to make it more flexible for the future.